src/client/package.json

```diff
@@ -92,51 +90,45 @@
     "@babel/preset-typescript": "^7.13.0",
     "@google-cloud/dialogflow": "^4.7.0",
     "@playwright/test": "^1.29.1",
-    "@storybook/addon-actions": "^7.0.2",
-    "@storybook/addon-essentials": "^7.0.2",
-    "@storybook/addon-links": "^7.0.2",
-    "@storybook/react": "^7.0.2",
-    "@storybook/react-vite": "^7.0.2",
-    "@testing-library/jest-dom": "^5.11.10",
-    "@testing-library/react": "^11.2.6",
-    "@testing-library/react-hooks": "^5.1.1",
-    "@testing-library/user-event": "^13.1.1",
+    "@storybook/addon-actions": "^6.5.13",
+    "@storybook/addon-essentials": "^6.5.13",
+    "@storybook/addon-links": "^6.5.13",
+    "@storybook/react": "^6.5.13",
+    "@testing-library/react": "14.0.0",
+    "@testing-library/user-event": "14.4.3",
     "@types/d3": "^6.3.0",
     "@types/google.analytics": "^0.0.42",
     "@types/lodash": "^4.14.178",
     "@types/node": "^14.14.37",
-    "@types/react": "^17.0.0",
-    "@types/react-dom": "^17.0.3",
+    "@types/react": "18.0.34",
+    "@types/react-dom": "18.0.11",
     "@types/react-helmet": "^6.1.1",
     "@types/react-measure": "^2.0.8",
-    "@types/react-router": "^5.1.14",
-    "@types/react-router-dom": "^5.1.7",
-    "@types/react-router-hash-link": "^2.4.4",
+    "@types/react-router": "5.1.20",
+    "@types/react-router-dom": "5.3.3",
     "@types/react-virtualized-auto-sizer": "^1.0.1",
     "@types/react-window": "^1.8.5",
-    "@types/styled-components": "^5.1.9",
-    "@typescript-eslint/eslint-plugin": "^5.45.0",
-    "@typescript-eslint/parser": "^5.45.0",
-    "@vitejs/plugin-react": "^3.0.0",
+    "@types/styled-components": "5.1.26",
+    "@typescript-eslint/eslint-plugin": "5.58.0",
+    "@typescript-eslint/parser": "5.58.0",
+    "@vitejs/plugin-react": "3.1.0",
     "concurrently": "^7.6.0",
     "cross-env": "^6.0.3",
-    "eslint": "^8.28.0",
-    "eslint-config-prettier": "^8.5.0",
-    "eslint-plugin-react": "^7.31.11",
+    "eslint": "8.38.0",
+    "eslint-config-prettier": "8.8.0",
+    "eslint-plugin-react": "7.32.2",
     "eslint-plugin-simple-import-sort": "^10.0.0",
     "husky": "^4.3.8",
     "jest-styled-components": "^7.1.1",
     "jsdom": "^21.1.1",
     "openfin-cli": "^4.0.0",
     "playwright": "^1.29.1",
-    "prettier": "2.3.0",
-    "react-test-renderer": "^17.0.2",
```

Comment:
thanks - this little varmint crept back in - watch out for stuff like this, @Dev-Alex23